### PR TITLE
[docs] Add a description for the JS configuration file

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,12 +119,38 @@ fixture `Electron test`
...
+    appArgs: ['--arg1', '--arg2'],
+    enableNavigateEvents: true
+}
+```
+
+The `.js` file must export an object with the configuration settings.
+
+Use JavaScript if you need additional flexibility. For instance, you can specify settings that depend on a condition.
+
+The browser provider supports the following options:
```
Comment (VasilyStrelyaev) [inline README.md:149]: Below this line formatting changes only.

File: README.md [hunk 2]
```diff
@@ -15,15 +15,25 @@ npm install testcafe-browser-provider-electron
 
 If your JavaScript application runs in Electron, follow these steps to set up testing.
 
-1. Create a `.testcafe-electron-rc` file in the root application directory. Include the following settings to this file.
+1. Create a JSON `.testcafe-electron-rc` or JavaScript `.testcafe-electron-rc.js` [configuration file](#configuration) in the root application directory. Include the following settings to this file.
```
Comment (AndreyBelym) [inline README.md:18]: We can also use the `.json` extension for JSON config files. Actually, I think we should always use an explicit extension, so we can add the extension everywhere in this file and do not mention extensionless config files as it is a legacy feature.

Comment (AndreyBelym) [reply README.md:18]:
```suggestion
1. Create a JSON `.testcafe-electron-rc.json` or JavaScript `.testcafe-electron-rc.js` [configuration file](#configuration) in the root application directory. Include the following settings to this file.
```

File: README.md [hunk 3]
```diff
@@ -108,12 +119,38 @@ fixture `Electron test`
 
 ## Configuration
 
-You can specify the following options in the `.testcafe-electron-rc` configuration file.
+You can specify browser provider options in a JSON `.testcafe-electron-rc` or JavaScript `.testcafe-electron-rc.js` configuration file.
```
Comment (AndreyBelym) [inline README.md:122]:
```suggestion
You can specify browser provider options in a JSON `.testcafe-electron-rc.json` or JavaScript `.testcafe-electron-rc.js` configuration file.
```

**Pull request merged**